deployment/network-operator/README.md

```diff
@@ -75,6 +75,14 @@ We have introduced the following Chart parameters.
 | `operator.tag` | string | `None` | Network Operator image tag, if `None`, then the Chart's `appVersion` will be used |
 | `deployCR` | bool | `false` | Deploy `NicClusterPolicy` custom resource according to provided parameters |
 
+### Proxy parameters
+
```

Comment:
after a bit of googling, it seems these proxy conventions are honored in both ubuntu and centOS so i dont think this needs to be separated per platform.

can you add a small paragraph explaining the these proxy parameters and what they are used for.

Reply:
added

Reply:
i would also add:

These proxy parameter will translate to `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` environment variables to be used by the network operator and relevant resources it deploys.